src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -9,7 +9,6 @@
     <Nullable>enable</Nullable>
     <IncludeInternalObsoleteAttribute>true</IncludeInternalObsoleteAttribute>
     <IsPackable>true</IsPackable>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
Did you want to prepare this for the PRs that @eiriktsarpalis has, or will that be a separate PR?

Reply:
I'd rather do that on every individual PR so that the commit contains the changes for historic reasons on what's needed to service.

Reply:
We have about 7-8 fixes in the pipeline, should we be doing this separately for every PR?

Reply:
Are all for JSON?

Reply:
A ok. Given that it is multiple fixes. I'll update this PR and adjust accordingly to build a new version of the JSON Package.

Reply:
Thanks 👍

Reply:
Done.

Reply:
I can't recall if we maintain a version for the ref-pack too or if we just let it sync with the ProductVersion.  Regardless, let's make sure that stays active in this release since I expect some changes will be in the source generator.

Reply:
Will do that in a separate PR.